server/src/core/error.ts

```diff
@@ -47,3 +48,7 @@ export function createResourceNotFoundError(message: string = ERROR_RESOURCE_NOT
 export function createUniquenessFailedError(message: string) {
   return createAppError(message, 'uniquenessFailed');
 }
+
+export function createNominationFailedError(message: string = ERROR_USER_CANNOT_BE_DONOR_AND_BENEFICIARY) {
+  return createAppError(message, 'nominationFailed');
+}
```

Comment:
I don't know if this should be the default error message for this nomination error. The default error message should be something obvious and generic for that type of error.

Reply:
When someone throws `createNominationFailedError` one would expect the error message to be something like "Nomination failed" or something like that. I think it would be better to have a function like `createBeneficiaryNominationFailedError` which creates a `nominationFailed` error with a default message like `Failed to nominate the specified user as a beneficiary` and then if you wanted a most situation-specific error like `user cannot be donor and beneficiary` you can pass that as an argument to the function.